docs/src/main/asciidoc/includes/security/providers/abac.adoc

```diff
@@ -119,9 +119,11 @@ Annotations: `@RolesAllowed`, `@RoleValidator.Roles`
 .Configuration example for `WebServer`
 ----
 security:
-  web-server.paths:
-    - path: "/user[/{*}]"
-      roles-allowed: ["user"]
+  features:
```

Comment:
This is wrong - either it can be in `security.web-server.paths`, or under `server.features...` - the root here is `security`

Reply:
aah.. yes... I missed it is under security and assumed it is the server.... My bad.